### DISCUSSION TITLE
Faulty micro created using cli, unable to delete it: Bad request id

### DISCUSSION THREAD

**Discussion by cashbackalarm** [deta/cloud-docs#737] [Help]
The cli seems to have created an faulty micro in my project.
All the fields under Details are empty, except memory showing "Upgraded!"
I am unable to to delete the micro using the frontend. I get a "Bad request id" error.
Any suggestion how to delete it?

**Comment by aavshr** [comment]
hi @cashbackalarm can you please provide the name/id of the micro if you can see it?

**Comment by cashbackalarm** [comment]
Hi Aavash,it is the micro named cdn in the cashbackalarm project.I cannot see any id for it.Kind regardsNadegeVon meinem iPhone gesendetAm 27.01.2023 um 21:00 schrieb Aavash Shrestha ***@***.***>:﻿
hi @cashbackalarm can you please provide the name/id of the micro if you can see it?
